### PR TITLE
Pass-through test

### PR DESCRIPTION
**Pull request by glazychev-art**
#521

### CODE REVIEW

File: pkg/networkservice/chains/nsmgr/server_test.go [hunk 1]
```diff
@@ -140,3 +148,114 @@ func TestNSMGR_LocalUsecase(t *testing.T) {
 	require.NotNil(t, conn2)
 	require.Equal(t, 5, len(conn2.Path.PathSegments))
 }
+
+func TestNSMGR_PassThrough(t *testing.T) {
```
Comment (denis-tingajkin) [inline pkg/networkservice/chains/nsmgr/server_test.go:152]: Can you also add a test that covers a case when all nses are registered in one nsmgr?

Comment (glazychev-art) [reply pkg/networkservice/chains/nsmgr/server_test.go]: Done

File: pkg/registry/common/interpose/server.go [hunk 2]
```diff
@@ -38,6 +38,11 @@ import (
 // interposeNSEName - a common prefix for all registered cross NSEs
 const interposeNSEName = "interpose-nse#"
 
+// GetCrossNSEPrefix - returns interposeNSEName
+func GetCrossNSEPrefix() string {
```
Comment (denis-tingajkin) [inline pkg/registry/common/interpose/server.go:42]: Can we just make `interposeNSEName` as public?

**Review by denis-tingajkin — COMMENTED** [review]
In general looks good to me

**Review by denis-tingajkin — APPROVED** [review]

File: pkg/networkservice/common/interpose/server.go [hunk 3]
```diff
@@ -135,10 +131,23 @@ func (l *interposeServer) Request(ctx context.Context, request *networkservice.N
...
 	id := ""
-	for i := conn.GetPath().GetIndex(); i > 0; i-- {
-		if conn.GetPath().GetPathSegments()[i].Name == l.name {
-			id = conn.GetPath().GetPathSegments()[i].Id
+	segmentIdx := conn.GetPath().GetIndex()
+	isCrossNSEPrev := false
+
+	l.endpoints.Range(func(key string, crossNSEURL *url.URL) bool {
+		isCrossNSEPrev =
+			conn.Path.PathSegments[segmentIdx-1].Name == strings.TrimPrefix(key, interpose.InterposeNSEName)
```
Comment (haiodo) [inline pkg/networkservice/common/interpose/server.go:139]: You should just check for Path segment Name to match endpoint name, you should not use interpose.InterposeNSENam it is just for registry.

**Review by haiodo — APPROVED** [review]

**Pull request merged**